files/pagerduty.rb

```diff
@@ -8,7 +8,11 @@ def region
   end
 
   def incident_key
-    "sensu #{region} #{@event['client']['name']} #{@event['check']['name']}"
+    if region && !region.empty?
+      "sensu #{region} #{@event['client']['name']} #{@event['check']['name']}"
```

Comment:
The reasoning here is that "datacenter" is the unique key for sensu clusters for the dashboard link. This means we can re-use this string as a unique identifier to dedupe *and* distinguish pagerduty alerts.

IMO "region" shouldn't show up in this codebase at all.